build_debian.sh

```diff
@@ -268,6 +268,10 @@ check system $HOST
   if memory usage > 50% for 5 times within 10 cycles then alert
   if cpu usage (user) > 90% for 5 times within 10 cycles then alert
   if cpu usage (system) > 90% for 5 times within 10 cycles then alert
+check process rsyslog with pidfile /var/run/rsyslogd.pid
+  start program = "/bin/systemctl start rsyslog.service"
+  stop program = "/bin/systemctl stop rsyslog.service"
+  if totalmem > 800 MB for 5 times within 10 cycles then restart
```

Comment:
>restart [](start = 57, length = 7)

Do we need to keep a restart counter somewhere?

Reply:
good question, what is the log message for such restart. we can search the syslog for such cases.

Reply:
Each cycle that monit detects the memory has exceeded the threshold it will log the following:

```
ERR monit[607]: 'rsyslog' total mem amount of 1.6 GB matches resource limit [total mem amount>800.0 MB]
```

And if it meets the criteria (5 of these withing 10 cycles), it will log the following when it attempts to restart the service:

```
INFO monit[607]: 'rsyslog' trying to restart
```